cfd_protocol/tests/dlc_protocol.rs

```diff
@@ -417,7 +417,13 @@ fn build_wallet<R>(
 where
     R: RngCore + CryptoRng,
 {
-    use bdk::{populate_test_db, testutils};
+    // TODO: Consider upstreaming these imports to be included in the macro.
```

Comment:
:rofl: I already did this: https://github.com/bitcoindevkit/bdk/pull/428

Reply:
Lucas, reading my thoughts before I have them, amazing!